src/tasks/studySession/channelReminder.js

```diff
@@ -29,12 +29,16 @@ export default function sendChannelReminder(client) {
         const mostRecentStudySessionMessage = studySessionMessages.first();
 
         if (isMessageOlderThan5HoursAgo(mostRecentStudySessionMessage)) {
-            mostRecentStudySessionMessage.delete().then(() => {
-                makeStudySessionMessage().then((studyMessage) => {
-                    studySessionChannel.send(studyMessage).catch((error) => {
+            makeStudySessionMessage().then((studyMessage) => {
+                if (upcomingSessionsAreDifferent(mostRecentStudySessionMessage, studyMessage)) {
+                    mostRecentStudySessionMessage.delete().then(() => {
+                        studySessionChannel.send(studyMessage).catch((error) => {
+                            console.log(error);
+                        });
+                    }).catch((error) => {
                         console.log(error);
                     });
-                });
+                }
             }).catch((error) => {
```

Comment:
With our current logic, it seems like we might be making a query to the database every minute after the 5 hours have passed until there is a change in session messages.
What do you think is the best way to handle this? I thought about changing the length of the interval, but that seems like it would get complicated.

Reply:
Ooh, yeah, you're right, I didn't notice that! I think there are two simple fixes:
1. Have an additional `cron.schedule` in scheduler.js that runs every five hours, and put this task in that instead of in the minute one 
2. Instead of checking if the post is 5 hours or more ago, check if the time between the post and now is exactly a multiple of 5 hours

Are either of these ok? Which one would you prefer?

Reply:
Smart! I like it a lot.